prog/minio/minio_cluster_nexus.rb

```diff
@@ -64,13 +72,30 @@ def before_run
   end
 
   label def wait
+    if !minio_cluster.root_certs.empty? && minio_cluster.certificate_last_checked_at < Time.now - 60 * 60 * 24 * 30 # ~1 month
```

Comment:
root_certs cannot be empty, right? Do we need the emptiness check? Is this for already created clusters? If so, it might be better to manually create certs for them and not add extra condition here.

Reply:
This is updated. I would like to move the existing cluster to an SSL enabled mode without changing the whole cluster. That's why I added a check to see if the certs are empty. If they are, I create certs for both fields and server does the same. This way, after the deployment we simply only need to install rhizome and the cluster will switch to ssl mode.

Reply:
Do you intend to keep this check after migrating the existing cluster?

Reply:
Thinking it further, I like the idea of executing it manually more. I will remove this code.